### ISSUE TITLE
Dockerapi-mailcow - server.py - high load CPU

### ISSUE THREAD

**Issue by xetorixik** [mailcow/mailcow-dockerized#1918]
**Describe the bug**
Dockerapi-mailcow, server.py generates a high load, up to 80-100% of the CPU.
Tested on an Mailcow upgrade and freshly installed system.
Both CentOS 7 and Debian 9.

**How to reproduce Reproduce**
Steps to reproduce the behavior please explain how to reproduce the bug:
1. Update Mailcow to the latest bits as of Oct 18, 2018.
2. Install a fresh system with the git bits from Oct 18, 2018.
3. The problem is immediately fixed when you switch back to git branch master as of Oct 15, 2018.
I chose this specific date, because on that day I knew for sure that it was working properly.

**Expected behavior**
The problem could be similar to https://github.com/mailcow/mailcow-dockerized/issues/1916
although I do not read anything of a high load. Merge both issues if you want.

**Logs**
The logs actually show pretty little, no errors apart from time over time requesting the sime (loop): 
what could cause a high load:

> dockerapi-mailcow_1  | 172.22.1.3 - - [19/Oct/2018 02:54:10] "GET /containers/json HTTP/1.1" 200 -
> dockerapi-mailcow_1  | 172.22.1.3 - - [19/Oct/2018 02:54:11] "GET /containers/json HTTP/1.1" 200 -
> dockerapi-mailcow_1  | 172.22.1.3 - - [19/Oct/2018 02:54:11] "GET /containers/json HTTP/1.1" 200 -
> dockerapi-mailcow_1  | 172.22.1.3 - - [19/Oct/2018 02:54:12] "GET /containers/json HTTP/1.1" 200 -
> dockerapi-mailcow_1  | 172.22.1.3 - - [19/Oct/2018 02:54:12] "GET /containers/json HTTP/1.1" 200 -
> dockerapi-mailcow_1  | 172.22.1.3 - - [19/Oct/2018 02:54:12] "GET /containers/json HTTP/1.1" 200 -
> dockerapi-mailcow_1  | 172.22.1.3 - - [19/Oct/2018 02:54:13] "GET /containers/json HTTP/1.1" 200 -
> dockerapi-mailcow_1  | 172.22.1.3 - - [19/Oct/2018 02:54:13] "GET /containers/json HTTP/1.1" 200 -
> dockerapi-mailcow_1  | 172.22.1.3 - - [19/Oct/2018 02:54:14] "GET /containers/json HTTP/1.1" 200 -
> dockerapi-mailcow_1  | 172.22.1.3 - - [19/Oct/2018 02:54:14] "GET /containers/json HTTP/1.1" 200 -
> dockerapi-mailcow_1  | 172.22.1.3 - - [19/Oct/2018 02:54:16] "GET /containers/json HTTP/1.1" 200 -

**System**
 - OS: Debian 9 and CentOS 7
 - Browser: Chrome, Opera.

Once again thanks @andryyy and team for the marvelous Mailcow.
Have a nice day.

**Comment by cklabautermann** [comment]
I can confirm this behaviour. after updating to the latest commit I see server.py eating up ca. 20 percent cpu on my server continuously.
OS: Ubuntu 18.04 minimal

Thanks,
Claudio

**Comment by altnd** [comment]
Good morning, i can reproduce it on my docker host  Mailcow is running on 4 Core Xeon with 16 GB Ram Ubuntu 18.04  
immediately after update to last commit the CPU load increases up to 30-50%.
Before upgrade the server load is round <2%. 


After checking the logs from docker-api container I can see the same logs as above: 

```

172.22.1.15 - - [19/Oct/2018 08:50:29] "GET /containers/json HTTP/1.1" 200 -
172.22.1.15 - - [19/Oct/2018 08:50:29] "GET /containers/json HTTP/1.1" 200 -
172.22.1.15 - - [19/Oct/2018 08:50:29] "GET /containers/json HTTP/1.1" 200 -
172.22.1.15 - - [19/Oct/2018 08:50:30] "GET /containers/json HTTP/1.1" 200 -
172.22.1.15 - - [19/Oct/2018 08:50:30] "GET /containers/json HTTP/1.1" 200 -
172.22.1.15 - - [19/Oct/2018 08:50:30] "GET /containers/json HTTP/1.1" 200 -
172.22.1.15 - - [19/Oct/2018 08:50:30] "GET /containers/json HTTP/1.1" 200 -
172.22.1.15 - - [19/Oct/2018 08:50:31] "GET /containers/json HTTP/1.1" 200 -
172.22.1.15 - - [19/Oct/2018 08:50:31] "GET /containers/json HTTP/1.1" 200 -
172.22.1.15 - - [19/Oct/2018 08:50:31] "GET /containers/json HTTP/1.1" 200 -
172.22.1.15 - - [19/Oct/2018 08:50:31] "GET /containers/json HTTP/1.1" 200 -
172.22.1.15 - - [19/Oct/2018 08:50:31] "GET /containers/json HTTP/1.1" 200 -
172.22.1.15 - - [19/Oct/2018 08:50:32] "GET /containers/json HTTP/1.1" 200 -
172.22.1.15 - - [19/Oct/2018 08:50:32] "GET /containers/json HTTP/1.1" 200 -
172.22.1.15 - - [19/Oct/2018 08:50:32] "GET /containers/json HTTP/1.1" 200 -
172.22.1.15 - - [19/Oct/2018 08:50:32] "GET /containers/json HTTP/1.1" 200 -

```

then check the .15 container on the network it is the acme container who produce the high load it's on a infinity loop to check autodiscover domain 


```
Fri Oct 19 08:54:50 CEST 2018 - Waiting for domain table... OK
Fri Oct 19 08:54:50 CEST 2018 - No A or AAAA record found for hostname autodiscover.
Fri Oct 19 08:54:50 CEST 2018 - Detecting IP addresses... OK
Validated CAA for parent domain mycldsrv.de
Fri Oct 19 08:54:51 CEST 2018 - Waiting for domain table... OK
Fri Oct 19 08:54:51 CEST 2018 - No A or AAAA record found for hostname autodiscover.
Fri Oct 19 08:54:51 CEST 2018 - Detecting IP addresses... OK
Validated CAA for parent domain mycldsrv.de
Fri Oct 19 08:54:51 CEST 2018 - Waiting for domain table... OK
Fri Oct 19 08:54:51 CEST 2018 - No A or AAAA record found for hostname autodiscover.
Fri Oct 19 08:54:51 CEST 2018 - Detecting IP addresses... OK
Validated CAA for parent domain mycldsrv.de
Fri Oct 19 08:54:51 CEST 2018 - Waiting for domain table... OK
Fri Oct 19 08:54:51 CEST 2018 - No A or AAAA record found for hostname autodiscover.
Fri Oct 19 08:54:51 CEST 2018 - Detecting IP addresses... OK
Validated CAA for parent domain mycldsrv.de
Fri Oct 19 08:54:51 CEST 2018 - Waiting for domain table... OK
Fri Oct 19 08:54:51 CEST 2018 - No A or AAAA record found for hostname autodiscover.
Fri Oct 19 08:54:51 CEST 2018 - Detecting IP addresses... OK
Validated CAA for parent domain mycldsrv.de
Fri Oct 19 08:54:52 CEST 2018 - Waiting for domain table... OK
Fri Oct 19 08:54:52 CEST 2018 - No A or AAAA record found for hostname autodiscover.
Fri Oct 19 08:54:52 CEST 2018 - Detecting IP addresses... OK
Validated CAA for parent domain mycldsrv.de
Fri Oct 19 08:54:52 CEST 2018 - Waiting for domain table... OK
Fri Oct 19 08:54:52 CEST 2018 - No A or AAAA record found for hostname autodiscover.
Fri Oct 19 08:54:52 CEST 2018 - Detecting IP addresses... OK
Validated CAA for parent domain mycldsrv.de
```

Thanks & have a nice day

**Comment by andryyy** [comment]
Thanks, that was indeed a very stupid mistake. Thank you guys and sorry. Fixed with e941c6e.

**Comment by cklabautermann** [comment]
looks good - thanks!
